src/app/components/hmcts-global-header/hmcts-global-header.component.ts

```diff
@@ -63,7 +63,7 @@ export class HmctsGlobalHeaderComponent implements OnInit, OnChanges {
     this.userDetails$ = this.appStore.pipe(select(fromAppStore.getUserDetails));
     this.isUserCaseManager$ = this.userDetails$.pipe(
       skipWhile((details) => !('userInfo' in details)),
-      map((details) => details.userInfo.roles),
+      map((details) => details.userInfo?.roles),
```

Comment:
Unlikely that you won't get userInfo, but not impossible
details?.userInfo?.roles